src/calva-fmt/src/infer.ts

```diff
@@ -97,15 +98,34 @@ function applyResults(r: ResultOptions, editor: vscode.TextEditor) {
     void editor
       .edit(
         (editBuilder) => {
+          if (isUndefined(r.edits)) {
+            console.error('Edits were undefined!', cloneDeep({ editBuilder, r, editor }));
+            return;
+          }
```

Comment:
As discussed, log an error and return if there are no edits because there is nothing to do if that is the case.